app/stores/UiStore.ts

```diff
@@ -324,6 +307,10 @@ class UiStore {
       theme: this.theme,
     };
   }
+
+  private persist = () => {
+    Storage.set(UI_STORE, this.asJson);
```

Comment:
Overall this solution is pretty dated, but we'll keep it for now for backwards compat